readme.md

```diff
@@ -296,6 +354,11 @@ unzip /tmp/data/berlin.zip -d /tmp/data
 
 # download openstreetmap data
 curl -s https://s3.amazonaws.com/metro-extracts.mapzen.com/berlin_germany.osm.pbf > /tmp/data/berlin.osm.pbf
+
+#download tiger data
+export TIGERPATH=/tmp/data/tiger #make sure that directory exists
```

Comment:
turns out i just forgot to update the doc when i switched the code to use pelias-config instead of env vars. thanks for pointing this out so i could revisit the doc and make the proper updates.